docs/resources/cluster.md

```diff
@@ -74,7 +74,7 @@ A resource for managing clusters on Instaclustr Managed Platform. A cluster cont
 |--------------------------|---------------------------------------|------------------------------------------------------------------------------------|
 | APACHE_CASSANDRA         | 2.2.18, 3.0.19, 3.11.8, 4.0 (preview) |                                                                                    |
 | SPARK                    | 2.1.3, 2.3.2                          | APACHE_CASSANDRA                                                                   |
-| KAFKA                    | 2.7.1, 2.8.2, 3.0.2, 3.1.2            |                                                                                    |
```

Comment:
We'll need a patch version bump in the makefile for these changes to kick in on the terraform registry docs

Reply:
Updated. Please have another look